lms/static/js/student_account/views/AccessView.js

```diff
@@ -1,19 +1,11 @@
 var edx = edx || {};
 
-(function($, _, _s, Backbone, gettext) {
+(function($, _, _s, Backbone, History) {
     'use strict';
 
     edx.student = edx.student || {};
     edx.student.account = edx.student.account || {};
 
-    // Bind to StateChange Event
-    History.Adapter.bind( window, 'statechange', function() {
```

Comment:
@bradenmacdonald why is this being removed? It makes the back button compatible with the logistration page.